### DISCUSSION TITLE
How to use nextTick() in Petite Vue project

### DISCUSSION THREAD

**Discussion by rguttersohn** [vuejs/petite-vue#119] [General]
I'm not understanding how to actually use nextTick() on a Petite Vue project. I can't seem to find any documentation for it. Any help would be great.

**Comment by halivert** [comment]
It works like in the standard Vue

https://codi.link/PCFET0NUWVBFIGh0bWw+CjxodG1sIGxhbmc9ImVuIj4KCTxoZWFkPgoJCTxtZXRhIGNoYXJzZXQ9InV0Zi04IiAvPgoJCTxtZXRhIG5hbWU9InZpZXdwb3J0IiBjb250ZW50PSJ3aWR0aD1kZXZpY2Utd2lkdGgiIC8+CgkJPHRpdGxlPk5leHQgVGljayB3aXRoIHBldGl0ZSB2dWU8L3RpdGxlPgogICAgPGxpbmsgaHJlZj0iaHR0cHM6Ly91bnBrZy5jb20vaGFsaXZlcnQuY3NzIiByZWw9InN0eWxlc2hlZXQiIC8+Cgk8L2hlYWQ+Cgk8Ym9keT4KCQk8bWFpbiBpZD0iYXBwIiB2LXNjb3BlPgoJCQk8aDE+Q291bnRlcjwvaDE+CgkJCTxwPgoJCQkJPGJ1dHRvbiBAY2xpY2s9ImFkZCI+U3VtIG9uZTwvYnV0dG9uPgoJCQk8L3A+CgkJCTxkaXY+e3sgY291bnRlciB9fSB7eyBuZXh0VGlja0NvdW50ZXIgfX08L2Rpdj4KCQk8L21haW4+Cgk8L2JvZHk+CjwvaHRtbD4K||aW1wb3J0IHsgY3JlYXRlQXBwIH0gZnJvbSAiaHR0cHM6Ly91bnBrZy5jb20vcGV0aXRlLXZ1ZT9tb2R1bGUiOwoKCQkJY3JlYXRlQXBwKHsKCQkJCWNvdW50ZXI6IDAsCgkJCQluZXh0VGlja0NvdW50ZXI6IDAsCgkJCQlhZGQoKSB7CiAgICAgICAgICBzZXRUaW1lb3V0KCgpID0+IHsKCQkJCQkgIHRoaXMuY291bnRlcisrOwogICAgICAgICAgfSwgNTAwKTsKCgkJCQkJdGhpcy4kbmV4dFRpY2soKCkgPT4gewoJCQkJCQl0aGlzLm5leHRUaWNrQ291bnRlcisrOwoJCQkJCX0pOwoJCQkJfQoJCQl9KS5tb3VudCgiI2FwcCIpOw==

~But I don't know if its intended that updates before the DOM rerenders >.<~ Yes, it's intended it works the same in the standard Vue.

**Reply by rguttersohn** [reply]
I see now. It's a built in method within CreateApp. Also, are you saying I can use nextTick to make updates before a re-render? Within a click event, I am trying to remove a class from a ref before some logic that decides whether it should be added again. However, I cannot get Petite Vue to remove and then add the class all within the same function.
